### PR TITLE
Move wiki into Readme

### PR DESCRIPTION
**Pull request by Gomez**
Moves the info from the wiki into readme plus adds info about Dovecot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,34 @@ Past contributors: [Jakob Sack](https://github.com/jakobsack), [Bart Visscher](h
 If you’d like to join, just go through the [issue list](https://github.com/owncloud/mail/issues) and fix some. :) We’re also in [#owncloud-mail on freenode IRC](https://webchat.freenode.net/?channels=owncloud-mail).
 
 
+## Troubleshooting
+
+### Gmail
+
+In case you are facing issues in accessing your Gmail account you can use https://accounts.google.com/DisplayUnlockCaptcha to unlock your account.
```
Comment (jancborchardt) [inline README.md]: Simplify: »If you can not access your Gmail account …«

File: README.md [hunk 2]
```diff
@@ -45,6 +45,34 @@ Past contributors: [Jakob Sack](https://github.com/jakobsack), [Bart Visscher](h
...
+
+### Outlook.com
+
+In case you are facing issues in accessing your Outlook.com account, you can try to enable the 'Two-Factor Verification' (https://account.live.com/proofs/Manage) and setup an app password (https://account.live.com/proofs/AppPassword), which you then use for the ownCloud mail app.
```
Comment (jancborchardt) [inline README.md]: Same here: »If you can not access your Outlook.com account …«

Also, »ownCloud Mail app«, Mail is capitalized. ;)

File: README.md [hunk 3]
```diff
@@ -45,6 +45,34 @@ Past contributors: [Jakob Sack](https://github.com/jakobsack), [Bart Visscher](h
...
+
+### Dovecot
+
+If your dovecot imap server prefixes all folders with `INBOX`, ownCloud Mail does not work correcty.
```
Comment (jancborchardt) [inline README.md]: »Dovecot IMAP« is the correct capitalization. ;)

File: README.md [hunk 4]
```diff
@@ -45,6 +45,34 @@ Past contributors: [Jakob Sack](https://github.com/jakobsack), [Bart Visscher](h
...
+
+Check `/etc/dovecot/dovecot.conf`:
+
+```
+namespace inbox {
+        separator = .
+        # All folders prefixed
+        # prefix = INBOX.
+        prefix =
+        inbox = ye
```
Comment (jancborchardt) [inline README.md]: Is this `ye` correct? Or is this a typo?

**Comment by jancborchardt** [comment]
Fine except the stuff I mentioned. :)

**Comment by Gomez** [comment]
Thanks! Fixed.

**Comment by jancborchardt** [comment]
@DeepDiver1975 can you remove the wiki? (Disable it in the repo settings.) Thanks!

**Comment by DeepDiver1975** [comment]
> @DeepDiver1975 can you remove the wiki? (Disable it in the repo settings.) Thanks!

done

**Pull request merged**